tests/CrawlerTest.php

```diff
@@ -197,8 +197,11 @@ public function profile_crawls_a_domain_and_its_subdomains()
 
         $profile = new CrawlInternalWithSubdomainUrls($baseUrl);
 
+        $protectedMethod = (new \ReflectionClass(CrawlInternalWithSubdomainUrls::class))->getMethod('isSubdomainOfHost');
```

Comment:
In this case I think I'd have the `isSubdomainOfHost` method public to improve the readability of the test but not sure 🤔